### PR TITLE
Extend suggestions (after #1407)

### PR DESCRIPTION
**Pull request by xitronix**

### CODE REVIEW

File: universal-login-react/src/ui/WalletSelector/Suggestions/KeepTypingSuggestion.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export const KeepTypingSuggestion = () => (
+  <ul className="suggestions-list">
+    <li className="suggestions-item">
+      <p className="suggestions-hint">This name is too short. Keep typing..</p>
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/WalletSelector/Suggestions/KeepTypingSuggestion.tsx]: There should be 3 dots

File: universal-login-react/src/core/utils/getSuggestionType.ts [hunk 2]
```diff
@@ -23,5 +26,5 @@ const isSingleCreation = (creations: string[], connections: string[]) =>
 const isSingleConnection = (creations: string[], connections: string[], actions: WalletSuggestionAction[]) =>
   !actions.includes(WalletSuggestionAction.recover) && creations.length === 0 && connections.length === 1;
 
-const isKeepTyping = (creations: string[], connections: string[]) =>
-  creations.length === 0 && connections.length === 0;
+const isNone = (creations: string[], connections: string[], source: string) =>
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/core/utils/getSuggestionType.ts]: I don't like`isNone`, maybe `isNoSuggestion` or something like that?

Comment (xitronix) [reply universal-login-react/src/core/utils/getSuggestionType.ts]: @cehali @rzadp ?

Comment (rzadp) [reply universal-login-react/src/core/utils/getSuggestionType.ts]: How about blank?
`SuggestionType = 'Blank'`, `const isBlank = ...`

Comment (Marik-D) [reply universal-login-react/src/core/utils/getSuggestionType.ts]: `Blank` sounds like there are suggestions but they are empty/plain, if that makes sense. I like the original name of `None`

**Comment by Marik-D** [comment]
@rzadp Force pushed rebase, pls re-aproove

File: universal-login-react/src/core/utils/getSuggestionType.ts (hunk 2 above)
Comment (Marik-D) [reply universal-login-react/src/core/utils/getSuggestionType.ts]: `NoSuggestions` / `isNoSuggestions`

Comment (marekkirejczyk) [reply universal-login-react/src/core/utils/getSuggestionType.ts]: Simplify and inline, see above #massivenamediscussion

File: universal-login-react/src/core/utils/getSuggestionType.ts [hunk 3]
```diff
@@ -7,14 +7,14 @@ export const getSuggestionType = (
   actions: WalletSuggestionAction[],
   source: string,
 ): SuggestionType => {
-  if (source.length < 3) {
+  if (source.length > 0 && source.length < 3) {
     return 'KeepTyping';
+  } else if (isNone(creations, connections, source)) {
```
Comment (marekkirejczyk) [inline universal-login-react/src/core/utils/getSuggestionType.ts]: #massivenamediscussion
```ts
...
} else if (source.length === 0) {
  return 'None';
} ...
```

**Pull request merged**